.github/workflows/docs.yml

```diff
@@ -6,10 +6,12 @@ on:
   pull_request:
     paths:
       - docs/**
+      - crates/pipeline_manager/src/**
```

Comment:
might want to set this to just src/pipeline_manager.rs as all the OpenAPI relevant parts are contained to this file anyway to try and keep CI time sanity

Reply:
nvm I guess this doesn't work as it includes types from other files :(

Reply:
Yeah exactly :(